css/fonts.css

```diff
@@ -47,26 +48,25 @@ h5{
 
 
 
-
 p{
 	font-size: 1.1em;
 	margin: 10px;
 	color: black;
+	font-family: arial;
 }
 
 
 
+
 p1{
```

Comment:
created the font for paragraphs to arial; test this version with arial and helvetica